src/io/flutter/FlutterStudioInitializer.java

```diff
@@ -7,9 +7,6 @@
 import com.intellij.openapi.project.Project;
 import org.jetbrains.annotations.NotNull;
 
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-
 public class FlutterStudioInitializer {
```

Comment:
Could you drop the changes to this file? I found a way to add the menu using XML so most of this is going away. But until I get further with New Module extensions I don't know if this class will be needed.